crates/indexed_docs/src/extension_indexed_docs_provider.rs

```diff
@@ -26,7 +26,6 @@ impl IndexedDocsProvider for ExtensionIndexedDocsProvider {
 
     fn database_path(&self) -> PathBuf {
         let mut database_path = PathBuf::from(self.extension.work_dir().as_ref());
-        database_path.push(self.extension.manifest().id.as_ref());
```

Comment:
Now that `Extension::work_dir` contains the extension's ID, we don't need to add it here.